python/private/py_console_script_binary.bzl

```diff
@@ -49,19 +49,19 @@ def py_console_script_binary(
     """Generate a py_binary for a console_script entry_point.
 
     Args:
-        name: str, The name of the resulting target.
-        pkg: target, the package for which to generate the script.
-        entry_points_txt: optional target, the entry_points.txt file to parse
+        name: [`target-name`] The name of the resulting target.
```

Comment:
nit: the related `docs/*.md` file has more useful user documentation that we may want to also move to the sphinx docs.

Reply:
Oh right, you added the merging functionality. This can be done using an include pretty easily.